src/features/pages/404-error/content/data.ts

```diff
@@ -0,0 +1,15 @@
+import { ButtonLink, InfoItem, ImageItem } from './types'
+import NotAvailableImage from 'images/svg/no_page_found.svg'
+
+export const not_available_image: ImageItem = {
+    src: NotAvailableImage,
+    alt: 'Page not found',
+}
+export const button_link: ButtonLink = {
+    text: '_t_Go home_t_',
+    url: { type: 'internal', to: '/' },
+}
+export const info_items: InfoItem = {
+    header: '_t_Oops, sorry..._t_',
+    sub_header: '_t_The requested URL was not found_t_',
+}
```

Comment:
please reuse existing types from `features/components/atoms` and follow ts rules 
```suggestion
export const not_available_image: IconProps = {
    src: NotAvailableImage,
    alt: 'Page not found',
}
export const home_link: LinkUrlType = {
    text: '_t_Go home_t_',
    url: { type: 'internal', to: '/' },
}
export const info_item: InfoItemType = {
    header: '_t_Oops, sorry..._t_',
    sub_header: '_t_The requested URL was not found_t_',
}
```

Reply:
You cant import from multiple files and not specific component type. @mohsen-deriv created the same 404 page where he created a separate type for that component and used..Thats the better approach